front/src/applications/operationalStudies/consts.ts

```diff
@@ -110,21 +110,20 @@ export const STUDY_STATES = {
 export type StudyState = keyof typeof STUDY_STATES;
 export const studyStates = Object.keys(STUDY_STATES) as StudyState[];
 
-const STUDY_TYPES = {
-  nothingSelected: 'nothingSelected',
-  timeTables: 'timeTables',
-  flowRate: 'flowRate',
-  parkSizing: 'parkSizing',
-  garageRequirement: 'garageRequirement',
-  operationOrSizing: 'operationOrSizing',
-  operability: 'operability',
-  strategicPlanning: 'strategicPlanning',
-  chartStability: 'chartStability',
-  disturbanceTests: 'disturbanceTests',
-};
+export const study_types = [
```

Comment:
I think it's better to keep it in SCREAM_CASE :) 
```suggestion
export const STUDY_TYPES = [
```